setup.py

```diff
@@ -23,6 +23,16 @@ def check_output(*popenargs, **kwargs):
 
 import sys
 
+if os.environ.get('COVERAGE','') == 'true':
```

Comment:
Updated, the environment variable can be any non-empty string. 

No, sir, `COVERAGE` is not used anywhere by the `coverage` module. But I think it would be better if we change it to - `MEASURE_COVERAGE`. Shall I change it?